JoinColony/colonyCDapp
# Handle Payment action

## Description

This PR adds support for the Payment action. It also contains a lot of wiring up that other actions will rely on.

<img width="972" alt="image" src="https://user-images.githubusercontent.com/112586815/219469179-faf31c3f-b7f8-4f5b-badc-9c34c576897b.png">

## Testing
1. Create a colony using colony creation flow.
2. Click on the Test Mint Tokens button. This should mint 5 tokens of your colony. You will be redirected to the transaction not found page. This is expected and is re-implemented in #261.
3. Go back and click on the Test Create Payment button. This should trigger a payment from the colony to your wallet of 2 tokens. You will again see the same page as above, refresh it a few times and you should see the action details page for the Payment action. Verify all details are shown as expected (the amount will be ~1.98 due to network fees).

The ingestor image should already be set up so that it runs the correct version, but if it's not picking the action try deleting the container in Docker Desktop and restarting your dev env:
![image](https://user-images.githubusercontent.com/112586815/219469528-9cabf7dd-8391-4a61-92f7-6d9231ee6f16.png)

Resolves #214
